### DISCUSSION TITLE
Looking for advice - You can only run on instance at SEB

### DISCUSSION THREAD

**Discussion by sehomer** [SafeExamBrowser/seb-win-refactoring#263] [Q&A]
Hello,

I installed SEB 3.3.1 on a virtual desktop (vm ware / win 10) and it won't start, saying "You can only run on instance at SEB". I did not change anything in the config file and tried allowing running in a vm (I think, SEB should at least start without that, stating that it's not allowed to run in a vm). Are there any hints or diagnostic steps to get this solved?

Best regards
Sebastian

**Comment by dbuechel** [comment]
SEB only allows to run one instance at a time. Thus, please check e.g. in the Windows task manager how many instances there are currently running.

**Reply by sehomer** [reply]
Hi Daniel,

there is only one instance running, but I'll check again. I startet a freshly installed Windows, installed SEB and started it once. Same after rebooting.

Best regards
Sebastian

**Comment by sehomer** [comment]
Hi Daniel,

I checked again, after loggin in there's only safeexambrowser.service running.

Best regards
Sebastian

**Reply by dbuechel** [reply]
SEB does run just fine with VDIs / VMs (especially by VMware) if allowed in the configuration, so there needs to be something on your system interfering with the mutex SEB uses to ensure that only a single instance is running at any given point in time.

**Comment by UNISTRAPHIOlivier** [comment]
@dbuechel Hello Dear, can you confirm that at this time SEB run on last vmware horizon environment in 3.8.0.742?
We encounter some issue.
